src/Central.php

```diff
@@ -152,7 +152,7 @@ public static function showGlobalView()
         if (Contract::canView()) {
             $grid_items[] = Contract::showCentral(false);
         }
-        if (Session::haveRight("logs", READ)) {
+        if (Session::haveRight(Log::$rightname, READ)) {
```

Comment:
Shouldn't it be the system log rights here instead ?

```suggestion
        if (Session::haveRight(Event::$rightname, READ)) {
```

Reply:
I think log permissions are OK here. In fact, I'm not sure there even needs to be a permission check since it only shows the events performed by the current user like when they logged in and which items they added/updated recently. Actually, I'm not sure this widget belongs on the Global view at all. It would fit in more on the Personal view I think.

Reply:
Indeed, I didn't realize it only shown personal informations.
With that in mind I agree on both points (no permission needed and should be in personal view)

Reply:
Probably best not to change this for now. I'm not convinced with the way the function works. A simple contains criteria for a user's username in the event message seems prone to false-matches.

Reply:
I agree to keep it like that.
@cconard96 Could you open a PR on GLPI 10.1 with the following changes:
1. Move this in personal view.
2. Remove the right checking.
3. Enhance the check to prevent false positives. For instance, we could use the following regex to get the matching entries: `(^{$username}|{$username}$|\s+{$username}\s+)`.